<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSlider" name="slider1">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>10</y>
     <width>31</width>
     <height>511</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget1" native="true">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>471</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="PlotWidget" name="graphView1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
